engine/security/src/main/scala/pl/touk/nussknacker/ui/security/oauth2/OAuth2Configuration.scala

```diff
@@ -42,4 +44,10 @@ object OAuth2Configuration {
   import net.ceedubs.ficus.readers.EnumerationReader._
 
   def create(config: Config): OAuth2Configuration = config.as[OAuth2Configuration](authenticationConfigPath)
-}
\ No newline at end of file
+}
+
+object ProfileFormat extends Enumeration {
```

Comment:
If I want to implement my own profile in extension - how will I deal with this configuration (it's mandatory)?

Reply:
THX, you're right. It should be optional.